source/fake/fake.go

```diff
@@ -19,13 +19,15 @@ package fake
 // Source implements FeatureSource.
 type Source struct{}
 
+// Name returns a friendly name for this source.
```

Comment:
Maybe change this description to `Name returns a identifier string for this feature source`?